src/sonic-yang-models/yang-models/sonic-device_neighbor_metadata.yang

```diff
@@ -0,0 +1,67 @@
+module sonic-device_neighbor_metadata {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-device_neighbor_metadata";
+    prefix device_neighbor_metadata;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description "DEVICE_NEIGHBOR_METADATA YANG Module for SONiC OS";
+
+    revision 2022-08-25 {
+        description "First Revision";
+    }
+
+    container sonic-device_neighbor_metadata {
+
+        container DEVICE_NEIGHBOR_METADATA {
+
+            description "DEVICE_NEIGHBOR_METADATA part of config_db.json";
+
+            list DEVICE_NEIGHBOR_METADATA_LIST {
+
+                key "name";
+
+                leaf name {
+                    type string {
+                        length 1..255;
+                    }
+                }
+
+                leaf lo_addr {
+                    type inet:ip-address;
+                }
+
+                leaf mgmt_addr {
+                    type inet:ip-address;
+                }
+
+                leaf hwsku {
+                    type stypes:hwsku;
+                }
+
+                leaf type {
+                    type string {
+                        length 1..255;
+                        pattern "ToRRouter|LeafRouter|SpineChassisFrontendRouter|ChassisBackendRouter|ASIC|MgmtToRRouter|SpineRouter|BackEndToRRouter|BackEndLeafRouter|EPMS|MgmtTsToR|BmcMgmtToRRouter|not-provisioned";
+                    }
+                }
+            }
+            /* end of list DEVICE_NEIGHBOR_METADATA_LIST */
+        }
+        /* end of container DEVICE_NEIGHBOR_METADATA */
+	}
```

Comment:
Wrong indentation level?

Reply:
Thanks for comments, fixed in latest code.